src/logger.js

```diff
@@ -31,0 +1,30 @@
+const chalk = require('chalk');
+
+const now = new Date();
+const time = chalk.dim(`${now.getMonth() + 1}/${now.getDate()} `) + chalk.cyan(`${now.getHours()}:${now.getMinutes()}:${now.getSeconds()}`);
+
+module.exports = {
+    /**
+   * 資訊
+   * @param {String} msg 訊息
+   */
+    info: function(msg) {
+        console.log(time + ' ' + chalk.blue('資訊') + ' ' + msg);
+    },

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
    },
```